UPTEthereumSigner/UPTEthSigner.m

```diff
@@ -64,7 +64,7 @@ + (void)signTransaction:(NSString *)ethAddress
     {
         NSError *protectionLevelError = [[NSError alloc] initWithDomain:@"UPTError"
                                                                    code:UPTSignerErrorCodeLevelParamNotRecognized.integerValue
-                                                               userInfo:@{@"message": @"protection level not found for eth address"}];
+                                                               userInfo:@{@"message": [NSString stringWithFormat:@"protection level not found for eth address: signTransaction %@", ethAddress]}];
```

Comment:
Line becomes much longer than 120 chars. Please introduce `let message = [NSString ...` at new line above and write `userInfo:@{ @"message" : message }` on this line. Note added spaces.